### PR TITLE
Changes subscribe button copy

### PR DESCRIPTION
**Pull request by joaojusto**
Changes subscribe button copy from `Get monthly updates` to `Get weekly updates` since we are sending a weekly newsletter.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/subvisual/blog.subvisual.co/pull/111" target="_blank">ReviewNinja

**Comment by naps62** [comment]
@joaojusto I fixed the build on master, so you can rebase to get the fix here

**Comment by joaojusto** [comment]
@naps62 done

File: app/views/shared/_newsletter.slim [hunk 1]
```diff
@@ -2,6 +2,6 @@ form action='//venturedeck.us5.list-manage.com/subscribe/post?u=79f7816bac08905f
   .Newsletter-input
     input.Input type='email' value='' placeholder='Your email address' name='EMAIL'
   .Newsletter-submit
-    input.ButtonPrimary.ButtonPrimary--fullWidth name='subscribe' type='submit' value='Get monthly updates'
+    input.ButtonPrimary.ButtonPrimary--fullWidth name='subscribe' type='submit' value='Get weekly updates'
```
Comment (zamith) [inline app/views/shared/_newsletter.slim]: This input does not have a label. Since we are changing it, we might as well add it.

**Comment by joaojusto** [comment]
@zamith done

File: app/views/shared/_newsletter.slim [hunk 2]
```diff
@@ -1,7 +1,8 @@
 form action='//venturedeck.us5.list-manage.com/subscribe/post?u=79f7816bac08905f81c1a8689&amp;id=f8b8f31221' method='post' id='mc-embedded-subscribe-form' name='mc-embedded-subscribe-form' class='Newsletter' target='_blank'
+  label.u-visuallyHidden for="mce-EMAIL"
```
Comment (zamith) [inline app/views/shared/_newsletter.slim]: Any particular reason for this not to be inside of `Newsletter-input`?

Comment (joaojusto) [inline app/views/shared/_newsletter.slim]: None, already changed the label to be inside newsletter-input

**Comment by zamith** [comment]
:+1:

**Pull request merged**